### ISSUE TITLE
Error with Progressbar

### ISSUE THREAD

**Issue by aniemerg** [avisingh599/visual-qa#19]
when running trainMLP.py, the training crashes in Keras' Progbar. The problem appears to be the following line 108: 
`progbar.add(args.batch_size, values=[("train loss", loss)])`

loss should probably be a float value of some sort, but when I look in the debugger, it is a list:
[array(7.346083164215088, dtype=float32)]

The following change appears to fix the issue:
`progbar.add(args.batch_size, values=[("train loss", loss[0].item(0))])`
